plugins/network-elements/nuage-vsp/pom.xml

```diff
@@ -28,6 +28,13 @@
     <version>4.6.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>${cs.lang3.version}</version>
+    </dependency>
+  </dependencies>
```

Comment:
Yes, this is used for the Pair class. Unfortunately Guava has no Pair class, or anything related, in their library.